spec/components/portfolios/details_component_spec.rb

```diff
@@ -67,6 +50,21 @@ def render_component(...)
   end
 
   before do
+    create(:program, parent: portfolio, status_code: status_code_a).tap do |program_a|
+      create(:project, parent: program_a, status_code: status_code_a).tap do |project_a|
+        create(:project, parent: project_a, status_code: status_code_b)
+      end
+    end
```

Comment:
Can't decide if this makes it better or worse
```suggestion
    create(:program, parent: portfolio, status_code: status_code_a).tap do |parent|
      create(:project, parent:, status_code: status_code_a).tap do |parent|
        create(:project, parent:, status_code: status_code_b)
      end
    end
```